ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1091 @@
+|The returned hostnames must be the same.
+
+|__http://localhost:8080/client/remote-outbound-notx-stateless__
+|Seven remote invocations of one stateless bean without a transaction context.
+The EJB client is expected to load balance the calls on various servers.
+The EJB remote call is constructed from the configuration of the `remote-outbound-connection`.
+|The list of the returned hostnames should contain occurrences of both`server2`
+and `server3`.
+
+|__http://localhost:8080/client/direct-stateless__
+|Two invocations under the transaction context started on `server1` (the sender side).
+The stateless bean is invoked on the remote side.
+The EJB remote call is constructed from the information defined directly
+in the application source code.
+The remote invocation is run, as all other calls of this quickstart, via the EJB remoting protocol.
+|The returned hostnames must be the same.
+
+|__http://localhost:8080/client/direct-stateless-http__
+|Two invocations under the transaction context started on `server1` (the sender side).
+The stateless bean is invoked on the remote side.
+The EJB remote call is constructed from the information defined directly
+in the application source code.
+The remote invocation is run, unlike the other calls of this quickstarts, via EJB over HTTP protocol.
+|The returned hostnames must be the same.
+
+|__http://localhost:8080/client/remote-outbound-notx-stateful__
+|Two invocations under the transaction context stared on `server1` (the sender side).
+Both calls are directed to the same stateful bean on the remote server because
+the stateful bean invocations are sticky by default.
+The EJB remote call is constructed from the configuration of `remote-outbound-connection`.
+|The returned hostnames must be the same.
+
+|__http://localhost:8080/client/remote-outbound-fail-stateless__
+|An invocation under the transaction context started on `server1` (the sender side).
+The call goes to one of the remote servers, where errors occur during transaction processing.
+But the failure happens when a two-phase commit decides about the work.
+You can see no error &ndash; the remote call finishes with success.
+This HTTP call finishes with success.
+Later it's the responsibility of the recovery manager to finish the work.
+|When the recovery manager finishes the work all the transaction resources are committed.
+
+|===
+
+=== Remote call failure invocation
+
+Let's put some more details for the failure case, when __http://localhost:8080/client/remote-outbound-fail-stateless__
+is invoked.
+
+This EJB call simulates the presence of an intermittent network error that
+occurs after the prepare phase of a JTA transaction but before the commit phase.
+
+The reason a two-phase commit protocol (2PC) is used is because there is more
+than one participant involved in the transaction: one for the database update
+and a second one (called `MockXAResource`) to force 2PC to be used. Since all
```

Comment:
```suggestion
and a second one (called `MockXAResource`) to force 2PC to be used. Because all
```

Our style guide recommends not using "since" as a synonym for 
"because" to avoid translation problems.